### PR TITLE
Imp: Drop support for marshmallow 2

### PR DESCRIPTION
**Pull request by lmignon**
depends on: https://github.com/marshmallow-code/marshmallow/pull/1429

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +22,9 @@ jobs:
       script: flake8 --show-source
       install: pip install flake8
       after_success: skip
```
Comment (SVilgelm) [inline .travis.yml]: You can remove `after_success`

File: ChangeLog [hunk 2]
```diff
@@ -1,6 +1,8 @@
 CHANGES
 =======
```
Comment (SVilgelm) [inline ChangeLog]: You don't need to modify change log manually, it will be generated by bumping a new tag

**Comment by lmignon** [comment]
@SVilgelm Thank you for the review. I'll finalyse this work into the next days. I'm also waiting for a fix on marshmallow itself https://github.com/marshmallow-code/marshmallow/pull/1429

**Comment by SVilgelm** [comment]
sure

**Comment by lmignon** [comment]
@SVilgelm Code fxed but still waiting for marshmallow-code/marshmallow#1429 (testsuite works with this fix on marshmallow...)

File: ChangeLog [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
 
+* Drop support for marshmallow 2
```
Comment (SVilgelm) [inline ChangeLog]: Please remove changes in this file. ChangeLoge will be generated automatically

**Comment by Sytten** [comment]
Any progress? I would love to this land soon

**Comment by lmignon** [comment]
@Sytten Unfortunately my PR on marshmallow remains without response... https://github.com/marshmallow-code/marshmallow/pull/1429. Can you review this PR? This may help to move things forward.

**Comment by lmignon** [comment]
@SVilgelm I implemented the propagation of the 'unknow' parameter at loading time as part of the marshmallow-object API to move forward with marshamallow-object...
This PR is now ready for review and merge...

File: ChangeLog (hunk 3 above)
Comment (SVilgelm) [reply ChangeLog]: ^^^ please revert. The bump version script automatically generates the change log file

**Pull request merged**